spikeinterface/toolkit/qualitymetrics/misc_metrics.py

```diff
@@ -139,23 +151,28 @@ def compute_isi_violations(waveform_extractor, isi_threshold_ms=1.5):
     seg_durations = [recording.get_num_samples(i) / fs for i in range(num_segs)]
     total_duraion = np.sum(seg_durations)
 
-    isi_threshold = (isi_threshold_ms / 1000. * fs)
+    isi_threshold_s = isi_threshold_ms / 1000
```

Comment:
You should the old stuff

Reply:
The formula works on seconds

Reply:
Yes it works but it force to have a vector multiplication.